src/index.ts

```diff
@@ -1,38 +1,48 @@
 #!/usr/bin/env node
 
+import program = require('commander');
```

Comment:
not seeing this in the package.json & lockfile above, please add

Reply:
updated